src/main/java/org/openhmis/manager/EnrollmentManager.java

```diff
@@ -72,17 +88,17 @@ public EnrollmentDTO getEnrollmentById(String enrollmentId) {
 		return enrollmentDTO;
 	}
 
-	public List<EnrollmentDTO> getEnrollments() {
+	public List<EnrollmentDTO> getEnrollments(EnrollmentSearchDTO searchDTO) {
 		List<EnrollmentDTO> enrollmentDTOs = new ArrayList<EnrollmentDTO>();
 
 		// Collect the inventories
-		List<TmpEnrollment> tempEnrollments = tmpEnrollmentDAO.getTmpEnrollments();
+		List<TmpEnrollment> tempEnrollments = tmpEnrollmentDAO.getTmpEnrollments(searchDTO);
 
 		// For each inventory, collect and map the data
 		for (Iterator<TmpEnrollment> iterator = tempEnrollments.iterator(); iterator.hasNext();) {
 			TmpEnrollment tempEnrollment = iterator.next();
 			EnrollmentDTO enrollmentDTO = EnrollmentManager.generateEnrollmentDTO(tempEnrollment);
-			enrollmentDTOs.add(enrollmentDTO);
+                        enrollmentDTOs.add(enrollmentDTO);
 		}
 		return enrollmentDTOs;
 	}
```

Comment:
This file also has an `getEnrollmentsByUpdateDate` method which is now obsolete and should be removed as part of this PR